.github/workflows/maven.yml

```diff
@@ -23,9 +23,9 @@ jobs:
           java-version: 11
 
       - name: Install Helm
-        uses: azure/setup-helm@v1
+        uses: azure/setup-helm@v3
```

Comment:
See: https://github.com/atlassian/data-center-helm-charts/actions/runs/3570398385
Somehow mock Kube is deemed to be 1.20 with this Helm version.